code/main.cpp

```diff
@@ -1,31 +1,31 @@
 #include <hwlib.hpp>
+#include <comm.hpp>
+
 #include <i2c_bus.hpp>
 #include <ssd1306_oled_buffered.hpp>
-int main(void) {
+
+#include "display_module.hpp"
+
+int main() {
+    using namespace r2d2;
```

Comment:
If you're 'using namespace' you might as well remove the namespace scoping below.